index.html

```diff
@@ -301,6 +301,22 @@ <h3>
           </table>
         </section>
 
+        <section id="normative">
+          <h3>
+            Registries
+          </h3>
+          <p>
+The Working Group may create a set of
+<a href="https://www.w3.org/2021/Process-20211102/#registries">registries</a>
+including <a href="https://www.w3.org/2021/Process-20211102/#registry-definition">registry definitions</a>
+and <a href="https://www.w3.org/2021/Process-20211102/#registry-table">registry tables</a>
+to support extension points in the above normative deliverables. Registries for
+extension points that are mandatory to use, for any of the above normative 
+deliverables (for example, Verifiable Credential properties that MUST be included in 
+a Verifiable Credential), must have at least one standardized entry.
```

Comment:
```suggestion
a Verifiable Credential), MUST have at least one standardized entry.
```

Reply:
> Not sure if this requires normative text to define so just wanted to point it out as an option.

W3C Charters typically do not use spec language. Doing so might raise a few eyebrows within the W3C AC during the vote. I suggest we stay away from [BCP14](https://www.rfc-editor.org/info/bcp14) language.